.travis.yml

```diff
@@ -32,6 +32,10 @@ notifications:
     # enables comments in the branch with the CI results
     webhooks: https://www.travisbuddy.com/
 
+before_deploy:
+  # Clean up all target dirs created by mvn inside the container so that the deploy script works (permission errors)
+  - docker exec -it test-container bash -v mvn -f /opt/openml/pom.xml clean
```

Comment:
If we issue a mvn clean, won't this erase the artifacts to be deployed?

Reply:
No, mvn deploy (done in the deploy.sh) does the entire lifecycle